ui/text_selection.js

```diff
@@ -111,11 +111,11 @@ shaka.ui.TextSelection = class extends shaka.ui.SettingsMenu {
   onCaptionStateChange_() {
     if (this.player.isTextTrackVisible()) {
       this.icon.textContent =
-          shaka.ui.Enums.MaterialDesignIcons.CLOSED_CAPTIONS_OFF;
+          shaka.ui.Enums.MaterialDesignIcons.CLOSED_CAPTIONS;
       this.button.ariaPressed = 'true';
     } else {
       this.icon.textContent =
-          shaka.ui.Enums.MaterialDesignIcons.CLOSED_CAPTIONS;
+          shaka.ui.Enums.MaterialDesignIcons.CLOSED_CAPTIONS__OFF;
```

Comment:
I think that the name is erroneous

Reply:
And that's why we have compilers and linters and tests!  :grin: 

@kartikeyshaurya, please take a look and try `python build/check.py` to confirm your changes.